opentuner/search/manipulator.py

```diff
@@ -222,7 +223,7 @@ class Parameter(object):
   """
   __metaclass__ = abc.ABCMeta
 
-  def __init__(self, name):
+  def __init__(self, name, **kwargs):
```

Comment:
A bunch of the classes extending Parameter allow **kwargs in their constructor and pass them up into the super class constructor. When this eventually gets to Parameter, I think it might cause an error since it's not expecting more args? (but this doesn't happen right now  since unknown args never get passed in)

I figured I should put it in for consistency.